src/main/java/org/sonarlint/intellij/actions/SonarLintToolWindow.java

```diff
@@ -469,5 +473,20 @@ private Content getTaintVulnerabilitiesContent() {
   @Override
   public void bindingChanged(@Nullable ProjectBinding previousBinding, @Nullable ProjectBinding newBinding) {
     runOnUiThread(project, this::refreshViews);
+
+    showSharedConfigurationNotification(project, "Project successfully bound with \"" +
```

Comment:
`bindingChanged` is applied to new binding, or to binding removal. I'd suggest to move everything to `ProjetBindingManager`, in the `bindTo` method instead

Reply:
You may have missed this one @eray-felek-sonarsource